roles/scale_object/node/tasks/yum/install.yml

```diff
@@ -14,3 +14,19 @@
    state: present
    disable_gpg_check: "{{ disable_gpgcheck }}"
   when: ansible_fqdn in scale_obj_nodes_list
+
+- name: install | Get installed spectrum-scale-object
+  shell: rpm -qa | grep spectrum-scale-object
+  register: scale_package_status
+  when: ansible_fqdn in scale_obj_nodes_list
+  ignore_errors: true
+  args:
+    warn: false
```

Comment:
Can you please add when condition to run this only rhel8 node

Reply:
`when: ansible_distribution in scale_rhel_distribution and ansible_distribution_major_version == '8'`